analytics/cdk/emr-eks/bin/emr-eks.ts

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env node
+import 'source-map-support/register';
+import * as cdk from 'aws-cdk-lib';
+import EmrEksStack from '../lib/emr-eks-stack';
+import { Stack } from 'aws-cdk-lib';
+import { EmrEksTeamProps } from '../lib/teams/emrEksTeam';
+import { PolicyStatement } from 'aws-cdk-lib/aws-iam';
+
+const app = new cdk.App();
+
+const account = '123456789012';
+const region = 'eu-west-1';
+
+const executionRolePolicyStatement: PolicyStatement[] = [
+  new PolicyStatement({
+    resources: ['*'],
+    actions: ['s3:*'],
+  }),
+  new PolicyStatement({
+    resources: ['*'],
+    actions: ['glue:*'],
+  }),
+  new PolicyStatement({
+    resources: ['*'],
+    actions: [
+      'logs:*',
+    ],
+  }),
+];
+
+const dataTeam: EmrEksTeamProps = {
```

Comment:
How can an existing VPC be used? I see logic added to the blueprint stack to pass in the user-provided VPC if it exists, but `eks-blueprints.VpcProvider` seems to take a string for the `vpcId`, and props.clusterVpc is specified as `IVpc`.
It would be good to add example code to show how to use an existing VPC.